src/libraries/System.Net.Http/tests/StressTests/HttpStress/build-local.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+## This is a helper script for non-containerized local build and test execution.
+## It downloads and uses the daily SDK which contains the compatible AspNetCore bits.
+## Usage:
+## ./build-local.sh [StressConfiguration] [LibrariesConfiguration]
+
+version=7.0
+repo_root=$(git rev-parse --show-toplevel)
+daily_dotnet_root=./.dotnet-daily
+
+stress_configuration="Release"
+if [ "$1" != "" ]
+then
+    stress_configuration=${1,,}                   # Lowercase all characters in $1
+    stress_configuration=${stress_configuration^} # Uppercase first character
```

Comment:
I don't think this really matter, I've been using lower-case interchangeably.

As a result, the whole if-else could be reduced to `stress_configuration="${1:-Release}"`

Reply:
Sometimes the scripts refer to filesystem locations based on the configuration variable values:

https://github.com/dotnet/runtime/blob/48a960215db417874803660e0667dce13df42dd5/src/libraries/System.Net.Http/tests/StressTests/HttpStress/build-local.sh#L38

Reply:
I learned the hard way it does matter in certain scenarios :rofl:. 
However, I was thinking how to make this more convenient to use, e.g.: `testhost_root=$(ls -t -d $repo_root/artifacts/bin/testhost/net$version-Linux-*-x64 | head -n 1)` which picks the most recent testhost dir if you do not specify which one you want to use.

So the script would work nicely without providing any argument with either release or debug testhost builds.

Reply:
BTW, this is just a suggestion for possible future improvement. I certainly won't hold the PR on this.

Reply:
For now I just made it fail with a verbose message if you build it with the wrong config.